Library/Formula/contacts-phonetic.rb

```diff
@@ -0,0 +1,12 @@
+class ContactsPhonetic < Formula
+  desc "Add phonetic names to contacts. Supports all CJK characters and multiple formats."
+  homepage "https://github.com/Elethom/ContactsUtil-PhoneticName"
+  url "https://github.com/Elethom/ContactsUtil-PhoneticName/archive/0.1.tar.gz"
+  version "0.1"
+  sha256 "c593ebfbedb3beac202f1de51189cf2b92c4a8d3b9da2fad633bb31f6b58e731"
+
+  def install
+    system "make", "install"
+    bin.install "contacts-phonetic"
```

Comment:
This seems redundant with the `make install`. Does `make install` not install `contacts-phonetic` to bin?